### ISSUE TITLE
Add support for DC.description

### ISSUE THREAD

**Issue by bondjimbond** [Islandora-Labs/islandora_social_metatags#6]

**Comment by rosiel** [comment]
Can we broaden this out? We have some "people" objects where we'd like to use MADS for the description, too. Maybe enable "profiles", like in solr metadata, so that datastreams and xpaths can be configured per content model?

**Comment by bondjimbond** [comment]
Good idea - I don't have time to work on it, but if someone wants to build I'd happily test. 

On the MADS specific piece. What MADS element would a person description typically map to? 
Side note - Looking at a person object in our own repository, I see there is a DC datastream, with extremely limited contents. A better mads-to-dc transform might be generally helpful... outside the scope of this module obviously.

**Comment by rosiel** [comment]
The IslandScholar site uses &lt;note type="history"/&gt; because... it was recommended by LOC! http://www.loc.gov/standards/mads/mads-name.xml though I think it could equally be &lt;note type="biographical/historical" /&gt; because of this page: http://www.loc.gov/standards/mads/mads-doc.html.

**Comment by rosiel** [comment]
Hey @bondjimbond  what do you think about adding support for solr fields as your source of content (instead of having to dive into xpaths in XML datastreams)?

**Comment by bondjimbond** [comment]
I'm game for it if I end up with time to do it, @rosiel (and/or time to figure it out)... I'm not likely to be finding time for a lot of Islandora dev work over the next month or two, so PRs are welcome. Want to open an issue for it?

If that's the preferred approach - would a config-per-content-model setup make the most sense, or would it be easier/better to allow multiple configurable fields, taking an if-not-then approach with the preferred field at the top? (e.g. [Islandora Simple Map](https://github.com/mjordan/islandora_simple_map))

**Comment by rosiel** [comment]
I'm in the middle of a refactor to have different configs per content model. Inspired (cough copy pasted) from solr metadata. I guess now I have to finish it!

**Comment by bondjimbond** [comment]
Hey, @rosiel - did you every get any further with this? (Obviously the release right now takes precedence of course; just curious.)

**Comment by rosiel** [comment]
@bondjimbond after the release! Sorry, I think I have a WIP on one of my builds, but can't find it at the moment.

**Comment by bondjimbond** [comment]
Hi @rosiel - just giving a prod to see whether this issue is still on your radar at all.
